client/shared/src/settings/temporary/TemporarySettings.ts

```diff
@@ -30,6 +30,7 @@ export interface TemporarySettingsSchema {
     'cta.browserExtensionAlertDismissed': boolean
     'cta.ideExtensionAlertDismissed': boolean
     'batches.defaultListFilters': MultiSelectState<BatchChangeState>
+    'batches.downloadSpecModal': boolean
```

Comment:
Maybe adding `Dismissed` to the end like a couple of the other examples do would make it clearer?

```suggestion
    'batches.downloadSpecModalDismissed': boolean
```

Reply:
I think this would also flip the default condition -- so it starts false (not dismissed) and should get set to true once it's dismissed.